reconcile/utils/parse_dhms_duration.py

```diff
@@ -31,6 +31,22 @@ def _seconds_to_seconds(seconds: int) -> int:
 }
 
 
+def seconds_to_hms(seconds: int) -> str:
+    s = seconds % 60
+
+    minutes = seconds // 60
+    if minutes == 0:
+        return f"{s}s"
+
+    m = minutes % 60
+    h = minutes // 60
```

Comment:
Can use `divmod` to save some lines:

```python
    minutes, s = divmod(seconds, 60)
    if minutes == 0:
        return f"{s}s"

    h, m = divmod(minutes, 60)
```

Do we need to support days (`d`) in it?